Automattic/wp-calypso
# LoTS: Introduce the assembler-first flow

Related to https://github.com/Automattic/wp-calypso/issues/83390

## Proposed Changes

* Introduce the assembler-first flow to move the Assembler before domains & plans.
* The feature is controlled by the `themes/assembler-first` flag
* I'll create a follow-up PR to display the launchpad for this flow after updating your homepage in the Site Editor

### User without any un-launched sites

https://github.com/Automattic/wp-calypso/assets/13596067/775d5170-edb9-4a65-8002-66fc018f3bb2

### Use with un-launched sites

https://github.com/Automattic/wp-calypso/assets/13596067/037bb09d-f748-41d4-8ebe-aafba236543a

## Testing Instructions



* Go to logged-out TS
* Select the Assembler CTA
* Sign up or Sign in
* If your account has an un-launched site, you will see “New or existing site”. Pick an existing site or create a new site.
* After the site selection or the site creation, you will see the Assembler immediately
* Finish the Assembler and ensure you land on the Site Editor

## Pre-merge Checklist
